src/DebuggerActions/FullStackDebugAction.class.st

```diff
@@ -45,6 +45,11 @@ FullStackDebugAction >> expandedStackSize [
 	^ 100000
 ]
 
+{ #category : #accessing }
+FullStackDebugAction >> help [
+	^ 'Open a full debuggeur to navigate in the full bugged stack.'
```

Comment:
"debuggeur" is it a typo or a french variant of word debugger? 
There is more than one such typo in this review, please consider to use english everywhere.

Reply:
Yes sorry! I'll push corrections. Thank you for the review.